<ui version="4.0" >
 <class>UiNewEditGroupBase</class>
 <widget class="QWidget" name="UiNewEditGroupBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>360</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>600</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>600</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>New / Edit Group</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../../resource/dms.qrc" >
    <normaloff>:/picture/32/images/32x32/group_32.png</normaloff>:/picture/32/images/32x32/group_32.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="2" >
    <widget class="QTreeWidget" name="treeWidgetGroup" >
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <column>
      <property name="text" >
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Description</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="labelGroupId" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>Group ID:</string>
       </property>
       <property name="buddy" >
        <cstring>lineEditGroupId</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditGroupId" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="frame" >
        <bool>true</bool>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btGroupId" >
       <property name="minimumSize" >
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="../../resource/dms.qrc" >
         <normaloff>:/picture/16/images/16x16/reload_16.png</normaloff>:/picture/16/images/16x16/reload_16.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="labelGroupName" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>Group name:</string>
       </property>
       <property name="buddy" >
        <cstring>lineEditGroupName</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditGroupName" >
       <property name="text" >
        <string/>
       </property>
       <property name="frame" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item rowspan="3" row="2" column="1" >
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QPushButton" name="btnAddGroup" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnUpdateGroup" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDeleteGroup" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="labelGroupName_2" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>Group description:</string>
       </property>
       <property name="buddy" >
        <cstring>lineEditGroupDescription</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditGroupDescription" >
       <property name="text" >
        <string/>
       </property>
       <property name="frame" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>464</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="2" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="btnOk" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>treeWidgetGroup</tabstop>
  <tabstop>lineEditGroupId</tabstop>
  <tabstop>btGroupId</tabstop>
  <tabstop>lineEditGroupName</tabstop>
  <tabstop>lineEditGroupDescription</tabstop>
  <tabstop>btnAddGroup</tabstop>
  <tabstop>btnUpdateGroup</tabstop>
  <tabstop>btnDeleteGroup</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources>
  <include location="../../resource/dms.qrc" />
 </resources>
 <connections/>
</ui>
